front/pages/api/poke/workspaces/[wId]/agent_configurations/[aId]/resurrect.ts

```diff
@@ -0,0 +1,79 @@
+import type { WithAPIErrorReponse } from "@dust-tt/types";
```

Comment:
Restore seems more appropriate than resurrect. Thinking about the day we will expose this in the product.

Reply:
Aligned